src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -103,7 +104,7 @@ public static File stringToHtml(String inxml, String infilexsl, String outfilena
 			File stylesheet = null;
 			File outfile = null;
 			StringReader inReader = new StringReader(inxml);
-			String tempOutfilename = outfilename + ".temp"; 
+			String tempOutfilename = outfilename + ".temp";

```

Comment:
Formatting changes should be done in a separate PR. https://github.com/zaproxy/zaproxy/wiki/DevGuidelines